docs/en/observability/logs-stream.asciidoc

```diff
@@ -143,10 +143,246 @@ If you're not seeing your log files in {kib}, check the following in the `elasti
+
+NOTE: These fields are part of the {ecs-ref}/ecs-reference.html[Elastic Common Schema (ECS)]. The ECS defines a common set of fields that you can use across Elasticsearch when storing data including log and metric data.
+
+See the following sections for information on extracting specific fields:
+- <<logs-stream-extract-timestamp>>
+- <<logs-stream-extract-loglevel>>
+- <<logs-stream-extract-hostip>>
+
+[discrete]
+[[logs-stream-extract-timestamp]]
+=== Extract the `@timestamp` field
+
+This section shows you how to extract the `@timestamp` field from the following example log:
+
+[source,log]
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+----
+
+To extract the timestamp you need to:
+- <<logs-stream-ingest-pipeline>>
+- <<logs-stream-index-template>>
+- <<logs-stream-create-data-stream>>
+
+[discrete]
+[[logs-stream-ingest-pipeline]]
+==== Use an ingest pipeline to extract the `@timestamp`
+
+To extract the `@timestamp` field, use an ingest pipeline with a dissect processor and a date processor. Processors are executed in the order they are defined, so in the following example: 
+
+. The {ref}/dissect-processor.html[dissect processor] extracts a structured field from your unstructured log message based on a pattern you set. In the following example, it extracts the timestamp to the `timestamp` field.
+. The {ref}/date-processor.html[date processor] then takes the `timestamp` field extracted by the dissect processor and extracts it to a date field. By default, this date field is called `@timestamp`. The date processor can also set the timezone, change the target field, and change the output format of the timestamp.
+
+The following example creates an ingest pipeline with a dissect and a date processor:
+
+[source,console]
+----
+PUT _ingest/pipeline/logs-example-default
+{
+  "description": "Extracts the timestamp from log",
+  "processors": [
+    {
+      "dissect": {
+        "field": "message",
+        "pattern": "%{@timestamp} %{message}"
+      }
+    },
+    {
+      "date": {
+        "field": "@timestamp",
+        "formats": ["ISO8601"]
+      }
+    }
+  ]
+}
+----
+
+. Set the following values for the dissect processor:
+- `field` – The field you're extracting data from, `message` in this case.
+- `pattern`– The pattern of the elements in your log data:
++
+[source,JSON]
+----
+%{timestamp} %{message}
+----
++
+This extracts the timestamp, `2023-08-08T13:45:12.123Z`, to the `timestamp` field, and the rest of the message, `WARN 192.168.1.101 Disk usage exceeds 90%.`, stays in the `message` field.
++
+. Set the following for the date processor:
+- `_ingest/pipeline/logs-example-default` – `logs-example-default` is the name of the pipeline. The name of your pipeline needs to match the name of your data stream. You'll set up your data stream in the next section. See {fleet-ref}/data-streams.html#data-streams-naming-scheme[data stream naming scheme] for more information.
```

Comment:
With just a dash between two code snippets, it's a little tough to see what you're saying to do here. Not sure how to improve this though. Maybe it's fine